docs/source/technical_tutorials/access/aws_access.ipynb

```diff
@@ -0,0 +1,965 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    prefix = "shared/smk0033/csv_ex/"
</pre><p>I suggest removing this variable completely and simply using the value directly in the call to s3.list_objects_v2 in the next cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/MAAP-Project/maap-documentation/pull/484/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='51'/>